docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
```

Comment:
This shouldn't expose to library's consumer in an official doc. this was for the repo maintainer